src/Exceptions/LivewireReleaseTokenMismatchException.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Livewire\Exceptions;
+
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+class LivewireReleaseTokenMismatchException extends HttpException
+{
+    public function __construct()
+    {
+        parent::__construct(
+            419,
+            "Livewire detected a release token mismatch. \n".
+            "This happens when Livewire's, the application's, or a component's release token has changed since this page was loaded."
```

Comment:
ask AI to help write this

Reply:
This happens when a user's browser session has been invalidated by a new deployment.